esx_service/vmdk_ops.py

```diff
@@ -715,18 +715,33 @@ def handleVmciRequests():
 
     lib.close(sock)  # close listening socket when the loop is over
 
+def usage():
+    print "Usage: %s -p <vSocket Port to listen on>" % sys.argv[0]
 
 def main():
     log_config.configure()
-    logging.info("=== Starting vmdkops service ===")
-
+    logging.info("=== Starting vmdkops service PID====")
     signal.signal(signal.SIGINT, signal_handler_stop)
     signal.signal(signal.SIGTERM, signal_handler_stop)
+    try:
+        port=1019
+        opts, args = getopt.getopt(sys.argv[1:], 'hp:')
```

Comment:
We do not ship that file by default. Also, ps will tell you the exact value vs. not being sure which version of config was loaded in..